src/views/main-menu.js

```diff
@@ -34,6 +34,17 @@ class MainMenu extends Component {
             width: '25vw'
           }}>Info Hub
           </button>
+          <button className="btn btn-warning" onClick={() => {
```

Comment:
please make this button btn-seconday

Reply:
But it needs to look different, I think

Reply:
In my opinion, the main menu should be coherent